src/app/zines/page.tsx

```diff
@@ -4,9 +4,16 @@ import { getPublishedZines } from "@/services/zine-service";
 export default async function Zines() {
 
   const zines = await getPublishedZines();
+  
+  if (!zines || zines.length === 0) {
+    return <p role="status">Não há zines publicados no momento.</p>;
+  }
```

Comment:
`zines?.length` avalia: se não há zines, retorna `undefined`.
Se há e o `length` é `0`, retorna `false`

```suggestion
  if (!zines?.length) {
    return <p role="status">Não há zines publicados no momento.</p>;
  }
```